shenyu-spring-boot-starter/shenyu-spring-boot-starter-client/shenyu-spring-boot-starter-client-motan/src/main/java/org/apache/shenyu/springboot/starter/client/motan/property/ShenyuMotanProperties.java

```diff
@@ -0,0 +1,246 @@
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.springboot.starter.client.motan.property;
+
+/**
+ * Shenyu motan properties.
+ */
+public class ShenyuMotanProperties {
+
+    private ProtocolConfig protocolConfig = new ProtocolConfig();
+
+    private BasicServiceConfig basicServiceConfig = new BasicServiceConfig();
+
+    /**
+     * Get the protocol config.
+     * @return protocol config
+     */
+    public ProtocolConfig getProtocolConfig() {
+        return protocolConfig;
+    }
+
+    /**
+     * Set the protocol config.
+     * @param protocolConfig protocol config
+     */
+    public void setProtocolConfig(final ProtocolConfig protocolConfig) {
+        this.protocolConfig = protocolConfig;
+    }
+
+    /**
+     * Get the BasicServiceConfig.
+     * @return basicServiceConfig
+     */
+    public BasicServiceConfig getBasicServiceConfig() {
+        return basicServiceConfig;
+    }
+
+    /**
+     * Set the basicServiceConfig.
+     * @param basicServiceConfig basicServiceConfig
+     */
+    public void setBasicServiceConfig(final BasicServiceConfig basicServiceConfig) {
+        this.basicServiceConfig = basicServiceConfig;
+    }
+
+    public static class ProtocolConfig {
+
+        private String name = "motan2";
+
+        private Integer maxContentLength = 1048576;
+
+        /**
+         * Get the name.
+         * @return name
+         */
+        public String getName() {
+            return name;
+        }
+
+        /**
+         * Set the name.
+         * @param name name
+         */
+        public void setName(final String name) {
+            this.name = name;
+        }
+
+        /**
+         * Get the maxContentLength.
+         * @return maxContentLength
+         */
+        public Integer getMaxContentLength() {
+            return maxContentLength;
+        }
+
+        /**
+         * Set the maxContentLength.
+         * @param maxContentLength maxContentLength
+         */
+        public void setMaxContentLength(final Integer maxContentLength) {
+            this.maxContentLength = maxContentLength;
+        }
+    }
+
+    public static class BasicServiceConfig {
+
+        private String export = "demoMotan:8002";
+
+        private String group = "motan-shenyu-rpc";
+
+        private boolean accessLog;
+
+        private boolean shareChannel = true;
+
+        private String module = "motan-demo-rpc";
+
+        private String application = "myMotanDemo";
```

Comment:
remove the default values of `application`, `module`, `export`.

Reply:
ok, i will fix it.